packages/core/src/storage/extract.ts

```diff
@@ -154,3 +159,23 @@ function loadLayoutType(varDecl: VariableDeclaration, layout: StorageLayout, der
     layout.types[type] = { label, members };
   }
 }
+
+function getDocumentationValues(varDecl: VariableDeclaration): [string | undefined, string | undefined] {
```

Comment:
The regular expressions here are not very robust. Please copy this code: https://github.com/OpenZeppelin/openzeppelin-transpiler/blob/744620735263f341f2c2a83ee5193d8e3011ff04/src/utils/extractNatspec.ts

And make this function return an object instead of an array to ensure we don't confuse the order of the return values.